index.js

```diff
@@ -46,11 +46,24 @@ class Plugin extends MiniAccountsPlugin {
     this._claimInterval = opts.claimInterval || util.DEFAULT_CLAIM_INTERVAL
     this._store = new StoreWrapper(opts._store)
     this._txSubmitter = createSubmitter(this._api, this._address, this._secret)
+    this._currencyScale = opts.currencyScale || 6
```

Comment:
You seem to assume that if `opts.currencyScale` is set, it is more than 6. Can you validate that the scale is higher than 6.

Alternatively, can you add tests that make sure everything works if the scale is < 6?

Reply:
👍 Also it should probably be `this._currencyScale = (typeof opts.currencyScale === 'number' ? opts.currencyScale : 6)` on the off chance that someone wants to use a scale of 0.

Reply:
I almost added a check to make sure it's more than 6, but decided less than 6 should be allowed. I'll add a test for that case though.

Reply:
Ok. Make sure to implement it the same way in `ilp-plugin-xrp-paychan` and `asym-client`.